cli/install/template.go

```diff
@@ -384,18 +384,23 @@ data:
       - source_labels: [__meta_kubernetes_namespace]
         action: replace
         target_label: namespace
+      # special case k8s' "job" label, to not interfere with prometheus' "job"
+      # label
+      # __meta_kubernetes_pod_label_conduit_io_proxy_job=foo =>
+      # k8s_job=foo
+      - source_labels: [__meta_kubernetes_pod_label_conduit_io_proxy_job]
+        action: replace
+        target_label: k8s_job
       # __meta_kubernetes_pod_label_conduit_io_proxy_deployment=foo =>
-      # k8s_deployment=foo
+      # deployment=foo
       - action: labelmap
```

Comment:
what does this action do when replacement is not specified?

Reply:
per https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config, `replacement` defaults to the matched group `$1`:
```
[ replacement: <string> | default = $1 ]
```